lib/kb_SPAdes/authclient.py

```diff
@@ -57,7 +60,7 @@ class KBaseAuth(object):
     A very basic KBase auth client for the Python server.
     '''
 
-    _LOGIN_URL = 'https://kbase.us/services/authorization/Sessions/Login'
+    _LOGIN_URL = 'https://kbase.us/services/auth/api/legacy/KBase/Sessions/Login'
```

Comment:
The old endpoint didn't work?

Reply:
Actually, mostly I'm surprised that this URL is still in there for so long.

Reply:
that's weird.  I didn't do that manually.  I just ran "make".  I wonder where it grabbed that URL from?

Reply:
manually fixed legacy URL (after testing that it was indeed make that added it).

Reply:
The url with legacy in it is the correct url. That comes from recompiling the client

Reply:
@MrCreosote wait... so I should keep https://kbase.us/services/auth/api/legacy/KBase/Sessions/Login ?

Reply:
Oh, and it's a default url - you can pass in a url in the constructor.

Reply:
yeah, the other url just redirects to that one in nginx. It doesn't *really* matter since it all winds up in the same place, and the server doesn't use the default url anyway.

Reply:
In any case, when you register it'll get recompiled, so it's confusing to have manual edits to generated code since they'll get blown away before deployment